src/main/java/org/sonarlint/intellij/core/BackendService.kt

```diff
@@ -158,11 +161,19 @@ class BackendService @NonInjectable constructor(private val backend: SonarLintBa
     }
 
     private fun toSonarQubeBackendConnection(createdConnection: ServerConnection): SonarQubeConnectionConfigurationDto {
-        return SonarQubeConnectionConfigurationDto(createdConnection.name, createdConnection.hostUrl, createdConnection.isDisableNotifications)
+        return SonarQubeConnectionConfigurationDto(
+            createdConnection.name,
+            createdConnection.hostUrl,
+            createdConnection.isDisableNotifications
+        )
```

Comment:
There are some changes like this one that are purely related to formatting. Are you following the style [guidelines](https://github.com/SonarSource/sonar-developer-toolset)? If so, I would suggest to move that to a separate commit